ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -1184,39 +1194,37 @@ private static XLMarker LoadMarker(IXLWorksheet ws, Xdr.MarkerType marker)
 
         #region Comment Helpers
 
-        private XDocument GetCommentVmlFile(WorksheetPart wsPart)
+        private static IList<XElement> GetCommentShapes(WorksheetPart worksheetPart)
         {
-            XDocument xdoc = null;
-
-            foreach (var vmlPart in wsPart.VmlDrawingParts)
+            // Cannot get this to return Vml.Shape elements
```

Comment:
It seems that because the VML file starts with an `xml` element, it can't be parsed with standard OpenXml, even though the inner elements are valid OpenXml elements, (e.g. `ShapeType` and `Shape`). I've even tried writing `xml`'s inner XML elements to a memory stream and using `OpenXMLReader` to parse that, but it still doesn't work. Maybe due to missing namespace declarations. It's a pity that we have to resort to returning `XElement`s.